src/Backend/Core/Engine/Model.php

```diff
@@ -389,7 +389,7 @@ public static function getExtrasForData(string $module, string $key, string $val
             array_filter(
                 $moduleExtras,
                 function (array $data) use ($key, $value) {
-                    $data = unserialize($data);
+                    $data = $data === null ? [] : unserialize($data);
```

Comment:
Didn't change anything to the logic, as the return value should be a bool ` return isset($data[$key]) && $data[$key] === $value;` (filter method)